### PR TITLE
Update Github SSO video to reflect RBAC changes and updates in Teleport 7

### PR DESCRIPTION
**Pull request by benarent**
Fixes https://github.com/gravitational/peopleware/issues/165

### CODE REVIEW

**Comment by russjones** [comment]
@benarent Is this ready to merge?

**Comment by benarent** [comment]
@russjones Yes, it's just waiting for approval

**Review by klizhentas — APPROVED** [review]

**Pull request merged**
